src/html/html-entity-finder.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {ASTNode} from 'parse5';
+import {Descriptor} from '../ast/ast';
+import {HtmlDocument, HtmlVisitor} from './html-document';
+import {EntityFinder} from '../entity/entity-finder';
+
+export interface HtmlEntityFinder
```

Comment:
yeah... I want an answer to the Entity/Descriptor question too. Let's go to the design doc and try to come with names for the resolved types too.